bitswap_test.go

```diff
@@ -102,27 +102,25 @@ func TestGetBlockFromPeerAfterPeerAnnounces(t *testing.T) {
 }
 
 func TestDoesNotProvideWhenConfiguredNotTo(t *testing.T) {
-	bitswap.ProvideEnabled = false
-	defer func() { bitswap.ProvideEnabled = true }()
-
+	bssession.SetProviderSearchDelay(50 * time.Millisecond)
```

Comment:
This is likely the source of the goroutines. See #17.

Reply:
Ah, good call, thanks. Fixed.